packages/renderer/src/lib/onboarding/Onboarding.svelte

```diff
@@ -265,6 +265,7 @@ async function updateOnboardingStep() {
 
 function setDisplayCancelSetup(display: boolean) {
   displayCancelSetup = display;
+  router.goto('/');
```

Comment:
Why this change?  This function updates the visibility of the dialog where the user will be asked to: 1) keep going on the onboarding 2) stop the onboarding and leave page. If the user click on the second there is already a `router.goto`

Reply:
That is correct observation. This goto is not required.
```suggestion
```